tests/src/Adapter/PredisAdapterTest.php

```diff
@@ -6,165 +6,9 @@
 /**
  * Integration test with Predis.
```

Comment:
Adapter tests are now should extend `AdapterTest`: https://github.com/everlution/redlock/pull/3/files#diff-4197cb1219aa0eeca5adc3f83597824aR3

Basically most content of this test has been moved to this Abstract test